Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -888,6 +890,7 @@ position=Position
 ppn=PPN
 priority=Priorit\u00E4t
 process=Vorgang
+processConfig.useProcessTemplateOcrdWorkflow=aus Produktionsvorlage
```

Comment:
I am not sure this is a good message key: the value does not contain any connection to OCR-D workflows, so the message key should probably reflect this? Perhaps something like `processConfig.fromTemplate` would be more appropriate? (we already have too many message keys were keys and values do not really match and I think when introducing new message keys we should try to do better)